lib/tool_shed/util/workflow_util.py

```diff
@@ -78,7 +78,7 @@ def get_data_inputs( self ):
         data_inputs = []
 
         def callback( input, value, prefixed_name, prefixed_label ):
-            if isinstance( input, galaxy.tools.parameters.DataToolParameter ):
+            if isinstance( input, galaxy.tools.parameters.basic.DataToolParameter ):
```

Comment:
Yes :). I will readily admit past-@jmchilton didn't understand the power or beauty of defining a module interface in the ``__init__.py`` file or have an conception of the the interface for parameters should be.

I would leave this the way it is for now - but at some point the parameters throughout Galaxy should be refactored so everything is accessed through ``galaxy.tools.parameters``.